commit e0a319b4b672c592f9824509d948914a4d167a1e
Author: wwp <subscript@free.fr>
Date:   Sat Aug 18 09:34:09 2018 +0200

    Fix for Bug 4072: Crash when clicking 'reply' or 'reply with quote'
    Wrong '%\n' format causes a crash in g_strdup_printf.

diff --git a/src/quote_fmt_parse.y b/src/quote_fmt_parse.y
index 03c872a67..92e300e03 100644
--- a/src/quote_fmt_parse.y
+++ b/src/quote_fmt_parse.y
@@ -491,7 +491,7 @@ static void quote_fmt_show_msg(MsgInfo *msginfo, const gchar *body,
 		while (fgets(buf, sizeof(buf), fp) != NULL) {
 			strcrchomp(buf);
 
-			if (!signature && account_signatures_matchlist_nchar_found(buf, "%\n"))
+			if (!signature && account_signatures_matchlist_nchar_found(buf, "%s\n"))
 				break;
 		
 			if (quoted && quote_str)
